src/Catel.MVVM/Catel.MVVM.Shared/Windows/Helpers/WrapControlHelper.cs

```diff
@@ -23,13 +22,15 @@ namespace Catel.Windows
     using System.Windows.Controls;
     using System.Windows.Controls.Primitives;
     using System.Windows.Data;
+
 #endif
 
     #region Enums
     /// <summary>
     /// Available wrap options that can be used in the <see cref="WrapControlHelper"/>.
     /// </summary>
     [Flags]
+    [ObsoleteEx(ReplacementTypeOrMember = "Catel.Windows.Serivce.WrapOptions", RemoveInVersion = "5.5")]
```

Comment:
TreatAsError = "5.0", RemoveInVersion = "6.0"

There is also a typo. Please just use Catel.Services instead of Catel.Windows.Services (especially if we are going to support other platforms).